pkg/controller/controller_binding.go

```diff
@@ -347,10 +347,23 @@ func isPlanBindable(serviceClass *v1alpha1.ServiceClass, plan *v1alpha1.ServiceP
 
 func (c *controller) injectBinding(binding *v1alpha1.Binding, credentials map[string]interface{}) error {
 	glog.V(5).Infof("Creating Secret %v/%v", binding.Namespace, binding.Spec.SecretName)
+	trueRef := true
+	ownerReference := metav1.OwnerReference{
+		APIVersion:         binding.APIVersion,
+		Kind:               binding.Kind,
```

Comment:
I do think it would be helpful to have an explanatory comment about why the `Controller` and `BlockOwnerDeletion` fields are set the way they are, since that is non-obvious to a casual reader.